src/Data/Ticket/LocalStorage/localStorageTicketDataClass.ts

```diff
@@ -135,6 +142,20 @@ async function attributes(): Promise<DataClassSchema> {
       'string',
       { title: lang === 'de' ? 'Referenznummer' : 'Reference number' },
     ],
+    responsibleAgent: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Zuständiger Agent' : 'Responsible agent',
```

Comment:
* [ ] Inconsistent wording: in other places we use `responsibleAgent: "Zuständiger Bearbeiter"`